### PR TITLE
Simple example vite

### PR DESCRIPTION
**Pull request by djhi**
Speeds up development dramatically. However, this won't work until we release the next version of Fakerest using modern bundlers (PR incoming)

### CODE REVIEW

File: examples/simple/index.html [hunk 1]
```diff
@@ -1,19 +1,14 @@
...
-</head>
-
-<body>
-    <div id="root">
-    </div>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite App</title>
```
Comment (fzaninotto) [inline examples/simple/index.html:6]: Please change

File: examples/simple/index.html [hunk 2]
```diff
@@ -1,19 +1,14 @@
 <!DOCTYPE html>
-<html>
-
-<head>
-    <title>React Admin</title>
-    <style>
-        body {
```
Comment (fzaninotto) [inline examples/simple/index.html:7]: This is required

File: examples/simple/package.json [hunk 3]
```diff
@@ -1,49 +1,26 @@
...
-        "ra-data-fakerest": "^3.9.0",
-        "ra-i18n-polyglot": "^3.9.0",
-        "ra-input-rich-text": "^3.9.0",
-        "ra-language-english": "^3.9.0",
-        "ra-language-french": "^3.9.0",
-        "react": "^17.0.0",
-        "react-admin": "^3.9.0",
-        "react-dom": "^17.0.0"
-    }
+  "name": "simple-vite",
```
Comment (fzaninotto) [inline examples/simple/package.json:2]: why change the name?

File: examples/simple/package.json [hunk 4]
```diff
@@ -1,49 +1,26 @@
...
+  "version": "0.0.0",
+  "scripts": {
+    "start": "vite",
```
Comment (fzaninotto) [inline examples/simple/package.json:5]: the dev script is used by codesandbox. Can you check that it can work with the vite version of the simple example?

**Review by fzaninotto — CHANGES_REQUESTED** [review]

File: examples/simple/package.json (hunk 4 above)
Comment (djhi) [reply examples/simple/package.json:6]: Ok codesandox will prefer `dev` over `start` but will use `start` if `dev` is not defined. However as I was unable to make Vite work on codesandbox, I added a webpack setup and a `dev` command

**Comment by fzaninotto** [comment]
Needs rebase

File: examples/simple/index.html [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    <title>React Admin</title>
+    <style>
+      body {
+        margin: 0;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script>window.global = window</script>
```
Comment (fzaninotto) [inline examples/simple/index.html:15]: why do we need this for webpack?

File: examples/simple/index-webpack.html [hunk 6]
```diff
@@ -1,19 +1,19 @@
 <!DOCTYPE html>
-<html>
-
-<head>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (fzaninotto) [inline examples/simple/index-webpack.html:5]: Why did you change that?

**Review by fzaninotto — CHANGES_REQUESTED** [review]

File: examples/simple/index.html (hunk 5 above)
Comment (djhi) [reply examples/simple/index.html:15]: This is needed for one of our dependency, can't remember which one though. Not related to webpack

File: examples/simple/index-webpack.html (hunk 6 above)
Comment (djhi) [reply examples/simple/index-webpack.html:5]: Because the old values didn't provide any value AFAIK. Besides, `minimum-scale` can be an accessibility issue for browsers that still respect them (those I use don't actually)

**Review by fzaninotto — APPROVED** [review]
